### PR TITLE
Modify Multiple Rules(IAC): Remove Non-IAC-Based Standards in Metadata (APPSEC-5)

### PR DESCRIPTION
**Pull request by loris-s-sonarsource**
Related to [**APPSEC-5**](https://sonarsource.atlassian.net/browse/APPSEC-5).

### CODE REVIEW

**Review by pedro-oliveira-sonarsource — CHANGES_REQUESTED** [review]
Just have a look at this PCI DSS node overwrite.

File: rules/S4423/cloudformation/metadata.json [hunk 1]
```diff
@@ -2,11 +2,28 @@
...
+    "OWASP Mobile": [
+    ],
+    "MASVS": [
+    ],
+    "OWASP Top 10 2021": [
+    ],
+    "PCI DSS 3.2": [
+    ],
+    "PCI DSS 4.0": [
+    ],
```
[the remark this answers is not in the document]
Comment (loris-s-sonarsource) [reply rules/S4423/cloudformation/metadata.json:24]: Oh, good point

Comment (loris-s-sonarsource) [inline rules/S4423/cloudformation/metadata.json:24]:
```suggestion
    "PCI DSS 3.2": [
      "4.1",
      "6.5.4"
    ],
    "PCI DSS 4.0": [
      "4.2.1",
      "6.2.4"
    ],
```

File: rules/S4423/terraform/metadata.json [hunk 2]
```diff
@@ -4,10 +4,27 @@
...
+    "OWASP Mobile": [
+    ],
+    "MASVS": [
+    ],
+    "OWASP Top 10 2021": [
+    ],
+    "PCI DSS 3.2": [
+    ],
+    "PCI DSS 4.0": [
+    ],
```
Comment (loris-s-sonarsource) [inline rules/S4423/terraform/metadata.json:26]:
```suggestion
    "PCI DSS 3.2": [
      "4.1",
      "6.5.4"
    ],
    "PCI DSS 4.0": [
      "4.2.1",
      "6.2.4"
    ],
```

File: rules/S5332/cloudformation/metadata.json [hunk 3]
```diff
@@ -1,8 +1,26 @@
...
+    "OWASP Mobile": [
+    ],
+    "MASVS": [
+    ],
+    "OWASP Top 10 2021": [
+    ],
+    "PCI DSS 3.2": [
+    ],
+    "PCI DSS 4.0": [
+    ],
```
Comment (loris-s-sonarsource) [inline rules/S5332/cloudformation/metadata.json:22]:
```suggestion
    "PCI DSS 3.2": [
      "4.1",
      "6.5.4"
    ],
    "PCI DSS 4.0": [
      "4.2.1",
      "6.2.4"
    ],
```

**Review by pedro-oliveira-sonarsource — APPROVED** [review]
LGTM.

**Pull request merged**